src/pages/patientView/PatientViewPage.tsx

```diff
@@ -43,6 +45,7 @@ interface IPatientViewState {
     mutationData: any;
     mrnaExprRankData?: MrnaRankData;
     activeTabKey: number;
+    hotspotsData: any;
```

Comment:
@alisman @adamabeshouse : Btw, we need this additional `HotspotData` mapping just because we don't have a dynamic mutation model. If it was possible to just attach the hotspot information to the mutation itself, say like `mutation.isHotspot = true;` then we wouldn't need to create such an additional data mapping.